internal/schema/schema_test.go

```diff
@@ -873,3 +873,137 @@ Second line of the description.
 		})
 	}
 }
+
+func TestInterfaceImplementsInterface(t *testing.T) {
+	for _, test := range []struct {
+		name                  string
+		sdl                   string
+		useStringDescriptions bool
```

Comment:
It seems that `useStringDescriptions` is not really used in your tests, so you could just hard-code the value below. It's just noise here.